src-ui/components/MultiScreen.cpp

```diff
@@ -181,6 +181,11 @@ void MultiScreen::layout()
 
 void MultiScreen::onVoiceInfoChanged() { sample->repaint(); }
 
+void MultiScreen::updateSamplePlaybackPosition(int64_t samplePos)
+{
+    sample->updateSamplePlaybackPosition(samplePos);
```

Comment:
No need for if sample here? I guess not!

Reply:
Since it is created in the MultiScreen Ctor and never changed I don't think it is necessary. Same as in onVoiceInfoChanged.